docs: add documentation for supporting AsyncAPI (#582)
asyncapi/community: PROJECT_FINANCE.md

> # Support AsyncAPI Initiative

Reviewer: ```suggestion
## Support AsyncAPI Initiative
```

> Your support would be greatly appreciated for AsyncAPI, where we aim to improve the current state of Event-Driven Architectures (EDAs) and their tooling.

Reviewer: ```suggestion
Your financial support would be greatly appreciated for AsyncAPI, where we aim to improve the current state of Event-Driven Architectures (EDAs) and their tooling.
```

> Your financial support would be greatly appreciated for AsyncAPI, where we aim to improve the current state of Event-Driven Architectures (EDAs) and their tooling.

Reviewer 2: ```suggestion
Your financial support would be greatly appreciated for the AsyncAPI Initiative, where we aim to improve the current state of Event-Driven Architectures (EDAs) and their tooling.
```

> In order to support AsyncAPI, the following options are available -

Reviewer 2: ```suggestion
In order to support AsyncAPI, the following options are available:
```

> - **Open Collective**: Visit our [Open Collective Page](https://opencollective.com/asyncapi)
> - **GitHub Sponsors**: Visit our [GitHub Sponsors Page](https://github.com/sponsors/asyncapi)
> - **Linux Foundation Crowdfunding**: Visit our [Linux Foundation Crowdfunding Page](https://crowdfunding.lfx.linuxfoundation.org/projects/445898e9-42a2-4965-9e0a-c2a714f381bc)

Reviewer 2: ```suggestion
- **Open Collective**: Visit our [AsyncAPI Open Collective page](https://opencollective.com/asyncapi)
- **GitHub Sponsors**: Visit our [AsyncAPI GitHub Sponsors page](https://github.com/sponsors/asyncapi)
- **Linux Foundation Crowdfunding**: Visit our [AsyncAPI Linux Foundation Crowdfunding page](https://crowdfunding.lfx.linuxfoundation.org/projects/445898e9-42a2-4965-9e0a-c2a714f381bc)
```

Improved links for accessibility.

Reviewer 2: this hasn't been fixed

> # How do we manage expenses?

Reviewer: ```suggestion
## How do we manage expenses?
```

1. AsyncAPI Initiative welcomes support in all 3 ways mentioned above.

> 2. In case of GitHub Sponsors, the fund is directly transferred to Open Collective.

Reviewer 2: ```suggestion
2. In the case of GitHub Sponsors, the fund is directly transferred to our AsyncAPI Open Collective.
```

> 3. Since Open Collective and Linux foundation Crowdfunding are totally disjoint, 
>    thus we can conclude that we can conclude that we manage expensed in 2 platforms ->

Reviewer 2: ```suggestion
3. Since Open Collective and Linux foundation Crowdfunding are disjointed, we manage expenses in 2 platforms:
```

Reviewer 2: this still hasn't been fixed

>    - Open Collective & GitHub Sponsors fundings on Open Collective

Reviewer 2: ```suggestion
   - Open Collective & GitHub Sponsors fundings transfers to Open Collective
```

>    - Linux Foundation Crowdfunding on the same 

Reviewer 2: ```suggestion
   - Linux Foundation Crowdfunding 
```
